components/result-count-title/src/index.js

```diff
@@ -4,16 +4,19 @@ import styled from 'react-emotion';
 
 import ResultCount from '@govuk-frederic/result-count';
 
-const Wrapper = styled('div')({
-  display: 'inline-block',
-});
+const Wrapper = styled('div')(
+  {
+    display: 'flex',
+    flexWrap: 'nowrap',
+  },
+);
 
 const Title = styled('div')({
   display: 'inline-block',
   fontFamily: 'Roboto, Arial, sans-serif',
   fontSize: '15px',
   lineHeight: '1.25',
-  marginLeft: '10px',
```

Comment:
removing this will change the appearance - reducing the spacing between the score and the title - this should probably be left as-is

Reply:
Thanks - this is a legacy thing from my CSS - I've reverted this.